### PR TITLE
Reorganize comments on sider

### PR DESCRIPTION
**Pull request by rohanshiva**
Remove comment type filter on sider. Comments are organized from oldest to latest (oldest on top). 

image

image

Will work on the toasts based on comment type in another PR.

### CODE REVIEW

File: client/src/index.css [hunk 1]
```diff
@@ -155,7 +155,7 @@ video {
   justify-content: flex-end;
   gap: 0.4rem;
   border-top: 0.1rem solid var(--border);
-  padding: 0.2rem 0.4rem;
+  padding: 0.6rem 0.4rem;
```
Comment (rohanshiva) [inline client/src/index.css:158]: image

Gives a bit more padding on top of the buttons on the context menu. 

Before:
image

Comment (Ramko9999) [reply client/src/index.css:158]: ty for the before and after

File: client/src/components/commentSider/CommentSider.tsx [hunk 2]
```diff
@@ -15,16 +14,13 @@ interface CommentProps {
...
   const { commentHighlight, highlightComment } = useContext(
     CommentHighlightContext
   );
   const profileBorderColor = getColor(author.username);
 
-  let className = "player-comment";
-  if (commentHighlight && commentHighlight.comment.id === id) {
-    className += " player-comment-selected";
-  }
+  const className = `player-comment ${(commentHighlight && commentHighlight.comment.id === id) && "player-comment-selected"}`
```
Comment (Ramko9999) [inline client/src/components/commentSider/CommentSider.tsx]:
```suggestion
const isHighlighted = commentHighlight && commentHighlight.comment.id === id;
  const className = `player-comment ${isHighlighted && "player-comment-selected"}`
```

File: client/src/components/commentSider/CommentSider.tsx [hunk 3]
```diff
@@ -51,74 +58,19 @@ function Comment({ comment }: CommentProps) {
 
 interface CommentListProps {
   comments: IComment[];
-  commentType: CommentType;
 }
 
-function CommentList({ comments, commentType }: CommentListProps) {
```
Comment (Ramko9999) [inline client/src/components/commentSider/CommentSider.tsx:55]: So happy to see all this nastiness erased.

**Review by Ramko9999 — APPROVED** [review]
1 small suggestion. Otherwise: 🚀

**Comment by Ramko9999** [comment]
Do you need to rebase? Looks like there are merge conflicts?

**Pull request merged**